### ISSUE TITLE
Last update 7205 or 7206?

### ISSUE THREAD

**Issue by VasasA** [backdrop-contrib/og#42]
The last implemented `hook_update_N()` is `og_update_7205()` in recommended D7 version 7.x-2.10
The last implemented `hook_update_N()` is `og_update_7206()` in development D7 version 7.x-2.x-dev

The `og_update_last_removed()` function returns `7206`. It causes error message, if the user wants to migrate a D7 site to Backdrop during update.php process, and the D7 site has used the recommended 7.x-2.10 version:
`og module can not be updated. Its schema version is 7205. Updates up to and including 7206 have been removed in this release. In order to update og module, you will first [need to upgrade](https://backdropcms.org/upgrade) to the last version in which these updates were available.`
![kép](https://user-images.githubusercontent.com/8567234/154594191-20f4e3a1-c1f7-4c49-a051-efdb0cc284af.png)

**I think the `7205` is the correct value of `og_update_last_removed()`.**

**Comment by laryn** [comment]
I ran into this previously in another module, and you are right, it does cause an upgrade error for people upgrading from the latest recommended version rather than the dev version.

We'll need to use the `7205` value and make sure that the fix for `7206` is included in the upgrade hook in a way that won't conflict if it's already been run on D7.

**Comment by laryn** [comment]
@VasasA Can you try to test that update path again? I think I have done it in a way that will work regardless of whether the `_7206` or `_7205` update was most recent on the site.

**Comment by VasasA** [comment]
- I have tested the upgrade path with OG **7.x-2.x** dev version. There wasn't error generated by `og_update_last_removed()`. The `lookup` index has been created in the database of Backdrop.
- I have tested the upgrade path with OG **7.x-2.10** version. There wasn't error generated by `og_update_last_removed()`. The `lookup` index has been created in the database of Backdrop.

It works :) Thank you very much. :+1:
